packages/lfo-input/src/LFOInput.ts

```diff
@@ -0,0 +1,188 @@
+import {
+  getNextEnumValue,
+  handleEachInput,
+  HedronEngine,
+  InputOptionNodesConfig,
+  IPlugin,
+  NodeValue,
+} from '@hedron/engine'
+
+const TAU = Math.PI * 2
+const lerp = (v0: number, v1: number, t: number) => (1 - t) * v0 + t * v1
+
+export class LFOInput implements IPlugin {
```

Comment:
This class looks pretty simple, It looks like a great example, doesn't feel like there is extraneous boilerplate. Feeling good about this system